webapp/components/experiments/EditExperiment.js

```diff
@@ -1,43 +1,221 @@
+  root: {
+    '& .editExperimentBack': {
+      display: 'flex',
+      cursor: 'pointer',
+      paddingLeft: theme.spacing(1),
+      '& .MuiTypography-root': {
+        marginLeft: theme.spacing(1),
+      },
     },
-    ],
-  };
+    '& .editExperimentBreadcrumb': {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      '& .MuiButton-startIcon': {
+        marginRight: theme.spacing(0.4),
+      },
+    },
+    '& .editExperimentAutocomplete': {
+      '& .MuiInputLabel-outlined.MuiInputLabel-shrink': {
+        transform: 'translate(14px, 9px) scale(0.75)',
+      },
+      '& .MuiAutocomplete-input': {
+        padding: '1rem 0.25rem 0.188rem',
+      },
+    },
+    '& .editExperimentList': {
+      display: 'flex',
+      marginBottom: theme.spacing(1),
+      width: '100%'
+    },
+    '& .MuiTypography-body2': {
+      fontSize: '1rem',
+    },
+    '& .editExperimentDefault': {
+      paddingLeft: theme.spacing(1),
+      overflow: 'auto',
+      maxHeight: '60vh'
+    },
+    '& .editExperimentHead': {
+      paddingLeft: theme.spacing(1),
+    },
+    '& .editExperimentGroup': {
+      borderRadius: theme.spacing(0.4),
+      padding: theme.spacing(2, 0),
+      '& .editExperimentBreadcrumb': {
+        paddingLeft: '0.625rem',
+      },
+      '& .editExperimentRow': {
+        paddingLeft: theme.spacing(4),
+        position: 'relative',
+        overflow: 'auto',
+        maxHeight: '25vh',
+        '&:before': {
+          background: '#4A4A4A',
+          content: '""',
+          height: '100%',
+          width: '0.125rem',
+          margin: 0,
+          display: 'block',
+          position: 'absolute',
+          left: '0.875rem',
+        },
+      },
+    },
+    '& .MuiAutocomplete-root': {
+      width: '100% !important',
+      '& .MuiAutocomplete-popper': {
+        marginTop: -theme.spacing(1),
+        '& .MuiAutocomplete-option': {
+          paddingLeft: theme.spacing(1),
+          paddingRight: theme.spacing(1),
+        }
+      },
+      '& .MuiPopover-root': {
+        '& .MuiPaper-root': {
+          '& .MuiList-root': {
+            '& .MuiMenuItem-gutters': {
+              paddingLeft: theme.spacing(2),
+              paddingRight: theme.spacing(2),
+            }
+          }
+        }
+      },
 
+      '& .MuiPopover-experiment': {
+        width: theme.spacing(14),
+      },
+    },
+    '& .MuiFormControl-root': {
+      width: '100%',
+    },
+    '& .MuiOutlinedInput-notchedOutline': {
+      border: '0 !important',
+    },
+    '& .MuiFilledInput-underline': {
+      '&:after, &:before': {
+        display: 'none',
+      },
+    },
+    '& .MuiFormLabel-root': {
+      fontWeight: 'normal',
+    },
+    '& .editExperimentFooter': {
+      position: 'absolute',
+      left: 0,
+      bottom: 0,
+      right: 0,
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      background: '#434343',
```

Comment:
that color is defined in `variables.less`& referenced in `@background_color_body_0` in `colors.less`
Make sure to reuse these definitions & please check for any hardcoded color in here